tests/modules/eventloop.c

```diff
@@ -160,12 +160,12 @@ int sanity(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
     }
     if (ValkeyModule_EventLoopDel(-1, VALKEYMODULE_EVENTLOOP_READABLE)
         == VALKEYMODULE_OK || errno != ERANGE) {
-        ValkeyModule_ReplyWithError(ctx, "ERR out of range fd should fail");
+        ValkeyModule_ReplyWithError(ctx, "ERR out-of-range fd should fail");
         goto out;
     }
     if (ValkeyModule_EventLoopDel(99999999, VALKEYMODULE_EVENTLOOP_READABLE)
         == VALKEYMODULE_OK || errno != ERANGE) {
-        ValkeyModule_ReplyWithError(ctx, "ERR out of range fd should fail");
+        ValkeyModule_ReplyWithError(ctx, "ERR out-of-range fd should fail");
```

Comment:
This is a user error message, so we should avoid changing it in case anyone has taken a character-by-character dependency on it.